Procfile

```diff
@@ -1,5 +1,5 @@
 release: bin/release
 web: bin/start-web ddtrace-run python -m gunicorn.app.wsgiapp -c gunicorn-prod.conf.py warehouse.wsgi:application
 web-uploads: bin/start-web ddtrace-run python -m gunicorn.app.wsgiapp -c gunicorn-uploads.conf.py warehouse.wsgi:application
-worker: env DD_SERVICE=warehouse-worker bin/start-worker ddtrace-run celery -A warehouse worker -Q default -l info --max-tasks-per-child 32
-worker-beat: env DD_SERVICE=warehouse-worker-beat bin/start-worker ddtrace-run celery -A warehouse beat -S redbeat.RedBeatScheduler -l info
+worker: bin/start-worker celery -A warehouse worker -Q default -l info --max-tasks-per-child 32
+worker-beat: bin/start-worker celery -A warehouse beat -S redbeat.RedBeatScheduler -l info
```

Comment:
Would it be cool to try this again with
```
worker: env DD_SERVICE=warehouse-worker DD_UNLOAD_MODULES_FROM_SITECUSTOMIZE=1 bin/start-worker ddtrace-run celery -A warehouse worker -Q default -l info --max-tasks-per-child 32
worker-beat: env DD_SERVICE=warehouse-worker-beat DD_UNLOAD_MODULES_FROM_SITECUSTOMIZE=1 bin/start-worker ddtrace-run celery -A warehouse beat -S redbeat.RedBeatScheduler -l info
```
by any chance? This should hopefully create more "isolation" between ddtrace and the celery application 🤞 .

Reply:
More details here: https://ddtrace.readthedocs.io/en/stable/configuration.html#DD_UNLOAD_MODULES_FROM_SITECUSTOMIZE

Reply:
We've shipped this in #14583 and will keep an eye on things over the next few hours.

Reply:
Same behavior. But we hadn't yet merged https://github.com/pypi/warehouse/pull/14585 so we were running 1.18.5, I noticed the env var you asked us to set in the release notes under known issues, does 1.18.5 respect this flag?

Reply:
Ah shame 🙁 . That env var was introduced around 1.8 so it's definitely respected in 1.18. Any logs that might give us a hint as to why the workers are timing out would be much appreciated (e.g. do they hang on SSL requests?)

Reply:
Unfortunately the workers just stop silently. We have the latest dd-trace out and running right now. I will try to make a close inspection if/when a worker stalls.

Reply:
Sorry. No more additional info was available in logs and we’re down to one worker now. A revert has been set on course for deployment, and there’s no obvious way for us to glean more information from running containers (run as non root user and we can’t add new utilities for inspection after deployment)